### DISCUSSION TITLE
Fuzzy matching

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [zauberzeug/nicegui#429] [Q&A] [answered]
Hello,
for my project, I am using an API that calls some products from the source.
I want to have fuzzy matching in input text fields.
How can I have that?
Thanks

**Comment by falkoschindler** [comment, marked as answer]
I'm not quite sure what you mean. Do you have something like #272 in mind?

**Reply by Shahin-rmz** [reply]
That is exactly what I meant.
#duplicate.
